web/src/app/components/login-page/login-page.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="card-container">
+  <mat-card class="card" fxFlex="30">
+    <mat-card-content fxLayout="column" fxLayoutAlign="space-between space-between" fxLayoutGap="20px">
+      <mat-card-title fxLayoutAlign="center">Sign into Dashboard</mat-card-title>
+      <button mat-flat-button class="github" >
+        <mat-icon svgIcon="github"></mat-icon>
+        <span>Sign in with Github</span>
+    </button>
+      <button mat-flat-button class="youtube">
+        <mat-icon svgIcon="youtube"></mat-icon>
+        <span>Sign in with Youtube</span>
+      </button>
+      <button mat-flat-button class="twitter">
+        <mat-icon svgIcon="twitter"></mat-icon>
+        <span>Sign in with twitter</span>
+        </button>
+      <button mat-flat-button class="instagram">
+        <mat-icon svgIcon="instagram"></mat-icon>
+        <span>Sign in with Instagram</span>
+      </button>
+    </mat-card-content>
+  </mat-card>
+</div>
```

Comment:
Missing blank line at the end of the file. The IDE should be add this automatically